docs/api-guide.md

```diff
@@ -607,6 +607,80 @@ Delete delete =
 transaction.delete(delete);
 ```
 
+#### Put and Delete with a condition
+You can write arbitrary conditions (e.g., a bank account balance must be equal to or more than zero) that you
+require a transaction to meet before being committed by having some checking logic in the transaction.
+Alternatively, you can also write simple conditions in a mutation operation, such as Put and Delete.
+
+When a Put or Delete operation includes a condition, it is only executed if the specified condition is met.
+If the condition fails to be satisfied when the operation is executed, an exception called `UnsatisfiedConditionException` is thrown.
+
+##### Conditions for Put
+You can specify a condition in a Put operation as follows:
+
+```java
+// Build a condition
+MutationCondition condition =
+    ConditionBuilder.putIf(ConditionBuilder.column("c4").isEqualToFloat(0.0F))
+        .and(ConditionBuilder.column("c5").isEqualToDouble(0.0))
+        .build();
+
+Put put =
+    Put.newBuilder()
+        .namespace("ns")
+        .table("tbl")
+        .partitionKey(partitionKey)
+        .clusteringKey(clusteringKey)
+        .floatValue("c4", 1.23F)
+        .doubleValue("c5", 4.56)
+        .condition(condition) // condition
+        .build();
+
+// Execute the Put operation
+transaction.put(put);
+```
+
+Other than the `putIf` condition, you can specify the `putIfExists` and `putIfNotExists` conditions as follows:
+
+```java
+// Build a putIfExists condition
+MutationCondition putIfExistsCondition = ConditionBuilder.putIfExists();
+
+// Build a putIfNotExists condition
+MutationCondition putIfNotExistsCondition = ConditionBuilder.putIfNotExists();
+```
+
+##### Conditions for Delete
+
+You can specify a condition in a Delete operation as follows:
+
+```java
+// Build a condition
+MutationCondition condition =
+    ConditionBuilder.deleteIf(ConditionBuilder.column("c4").isEqualToFloat(0.0F))
+        .and(ConditionBuilder.column("c5").isEqualToDouble(0.0))
+        .build();
+
+Delete delete =
+    Delete.newBuilder()
+        .namespace("ns")
+        .table("tbl")
+        .partitionKey(partitionKey)
+        .clusteringKey(clusteringKey)
+        .condition(condition)  // condition
+        .build();
+
+// Execute the Delete operation
+transaction.delete(delete);
+```
+
+Other than the `deleteIf` condition, you can specify the `deleteIfExists` condition as follows:
```

Comment:
Does this part mean "Instead of using the `deleteIf` condition"?

```suggestion
Instead of using the `deleteIf` condition, you can specify the `deleteIfExists` condition as follows:
```

Or does this part mean "In addition to using the `deleteIf` condition"?

```suggestion
In addition to using the `deleteIf` condition, you can specify the `deleteIfExists` condition as follows:
```

Reply:
Same thinking as for the Put operation here. Either a `deleteIf` or a `deleteIfExists` condition can be specified for a Delete operation so the `In addition to using...` suggestion is more appropriate, isn't it?

Reply:
@Torch3333 In that case, yes, let's use my second suggestion "In addition to using ... " here as well.